src/gpio.rs

```diff
@@ -11,6 +11,13 @@ pub trait GpioExt {
     fn split(self) -> Self::Parts;
 }
 
+trait GpioRegExt {
+    fn in_low(&self, pos: u8) -> bool;
+    fn out_low(&self, pos: u8) -> bool;
```

Comment:
Actually I meant changing `out_low` to `is_set_low` and `in_low` to `is_low` to match up with the digital trait names.

Reply:
Ah, fair. Will tweak.

Reply:
Done